cmd/src/gateway.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+)
+
+var gatewayCommands commander
+
+func init() {
+	usage := `'src gateway' manages Cody Gateway operations on a Sourcegraph instance.
```

Comment:
Since Cody Gateway doesn't run as part of a Sourcegraph instance, I think this phrasing is maybe a bit confusing?

Perhaps "interacts with Cody Gateway directly or through a Sourcegraph instance" may be better?